src/modules/referral/services/queries.ts

```diff
@@ -4,10 +4,10 @@ export const getReferralsQuery = () => {
       *,
       case
         when d."depositorAddr" = $1 and d."referralAddress" = $1
-          then cast(d."realizedLpFeeUsd" * (1 + d."referralRate") / 0.025 * power(10, 18) as varchar)
+          then cast(d."realizedLpFeeUsd" * (1 + d."referralRate") / $2 * power(10, 18) as varchar)
         when d."depositorAddr" = $1
-        then cast(d."realizedLpFeeUsd" * (1 + d."referralRate") / 0.025 * 0.25 * power(10, 18) as varchar)
-        else cast(d."realizedLpFeeUsd" * (1 + d."referralRate") / 0.025 * 0.75 * power(10, 18) as varchar)
+        then cast(d."realizedLpFeeUsd" * (1 + d."referralRate") / $2 * 0.25 * power(10, 18) as varchar)
```

Comment:
The price is not hardcoded anymore in the SQL query, but received as a dynamic parameter coming from the global config

Reply:
Why $2?